src/main/java/org/team1540/robot2025/subsystems/elevator/Elevator.java

```diff
@@ -104,21 +106,25 @@ public void holdPosition() {
         setPosition(inputs.positionMeters[0]);
     }
 
+    public void resetPosition(double positionMeters) {
+        io.resetPosition(positionMeters);
+    }
+
     public Command setpointCommand(ElevatorState state) {
```

Comment:
also missing a verb. I'm actually a big fan of `commandVerb` rather than `verbCommand` (although the more important thing is that it's standardized). That would let you have `commandSetSetpoint` or `commandToSetpoint`.

Reply:
Ok i was making the APIs consistent and just chose one. Why do you prefer the command prefix and not suffix?